tools/db_crashtest.py

```diff
@@ -269,6 +269,20 @@
     "low_pri_pool_ratio": lambda: random.choice([0, 0.5]),
     "soft_pending_compaction_bytes_limit" : lambda: random.choice([1024 * 1024] + [64 * 1073741824] * 4),
     "hard_pending_compaction_bytes_limit" : lambda: random.choice([2 * 1024 * 1024] + [256 * 1073741824] * 4),
+    "enable_sst_partitioner_factory": lambda: random.choice([0, 1]),
+    "enable_do_not_compress_roles": lambda: random.choice([0, 1]),
+    # TODO(hx235): enable `block_align` after fixing the surfaced corruption issue
+    "block_align": 0,
+    "lowest_used_cache_tier": lambda: random.choice([0, 1, 2]),
+    "enable_custom_split_merge": lambda: random.choice([0, 1]),
+    # TODO(hx235): enable other `adm_policy` after fixing the surfaced segfault issue with compressed secondary cache
+    "adm_policy": lambda: random.choice([0]),
+    "last_level_temperature": lambda: random.choice([0, 4, 8, 12]),
```

Comment:
Maybe we can leave a TODO to convert these integers into human readable names, like the `compression_type` argument is taking.

Reply:
Fixed with human readable names